### PR TITLE
feat: Add timezone picker

### LINKED ISSUE #4734: Scheduler: Display timezone on public event schedule
With the move to online events and people joining an event in different timezones displaying the timezone would be super helpful. Therefore, please:

1. Implement timezone info on top right on the public event pages.
2. In a second step add a drop down menu to change the timezone and display the correct event time and date in the timezone choosen by the user.

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #4734

#### Short description of what this resolves:
Display timezone on public event page

Screenshot from 2020-08-24 22-26-19
Screenshot from 2020-08-24 22-26-15
Screenshot from 2020-08-24 22-25-56

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
- [ ] It should say `Europe/Paris (Event)` and `Asia/Kolkata (Local)`
- [ ] Event timezone should be the default selected, not local timezone
- [ ] Local timezone should be the first option in dropdown

**Comment by maze-runnar** [comment]
Screenshot from 2020-08-24 23-20-03

File: app/controllers/public/sessions.js [hunk 1]
```diff
@@ -2,13 +2,17 @@ import classic from 'ember-classic-decorator';
...
 
 @classic
 export default class SessionsController extends Controller {
+
   queryParams = ['sort'];
   sort = 'starts-at';
   isTrackVisible = false;
-
+  timezones = timezones;
+  currentTimezone = moment.tz.guess();
```
Comment (iamareebjamal) [inline app/controllers/public/sessions.js:13]: This should be event timezone by default, not local timezone

Comment (maze-runnar) [reply app/controllers/public/sessions.js:13]: Session page drop down is showing event timezone by default, not local timezone

Comment (maze-runnar) [reply app/controllers/public/sessions.js:13]: > currentTimezone = moment.tz.guess();

this line is for in other condition current timezone, other than event timezone

**Pull request merged**